ui/package.json

```diff
@@ -31,7 +31,7 @@
     "start:chroot": "ember server --proxy=http://127.0.0.1:8300 --port=4300",
     "test": "concurrently --kill-others-on-fail -P -c \"auto\" -n lint:js,lint:hbs,vault \"yarn:lint:js:quiet\" \"yarn:lint:hbs:quiet\" \"node scripts/start-vault.js {@}\" --",
     "test:enos": "concurrently --kill-others-on-fail -P -c \"auto\" -n lint:js,lint:hbs,enos \"yarn:lint:js:quiet\" \"yarn:lint:hbs:quiet\" \"node scripts/enos-test-ember.js {@}\" --",
-    "test:oss": "yarn run test -f='!enterprise' --split=8 --preserve-test-name --parallel",
```

Comment:
I know we're getting desperate here, so by all means if this works go for it. but removing parallel will make test run about 2x slower. I tried a similar tactic a bit back and pinged Matthew about the consequences. I'll send you the slack thread.

Reply:
Yeah, I think that's fine because now they've run twice and been successful. I haven't two successful test runs in a row in days and days 😅

Reply:
We still have the `test-quick` options for running tests locally which is great!